course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -85,33 +100,27 @@
                 },
             },
             lengthChange: false,
-            sAjaxDataProp: 'aaData',
+            sAjaxDataProp: 'data',
```

Comment:
Sadly, yes.  If you use the function version of the `ajax` argument, and you're using server-side processing (since that requires returning an object with the rowsFiltered/rowsReturned attrs, rather than just an array of data), your only option to make sure the data gets displayed is using the deprecated `sAjaxDataProp` param.

I found a datatables forum post where one of the devs confirmed it was a bug at one point, I'll see if I can scare it up for a comment.